lms/startup.py

```diff
@@ -24,6 +24,9 @@ def run():
     """
     Executed during django startup
     """
+    if settings.FEATURES.get('ENABLE_THIRD_PARTY_AUTH', False):
+        enable_third_party_auth()
+
```

Comment:
Whenever a certain ordering needs to be maintained for a reason, it's good practice to have a comment that states why and what it needs to come before. Could you please add one here?